apim/4.x/build/files/graviteeio-apim-portal-ui.conf

```diff
@@ -1,9 +1,45 @@
 server {
 	listen  0.0.0.0:8085;
-	root    /opt/graviteeio/apim/portal-ui;
-	index index.html;
 
-	location / {
-		try_files $uri $uri/ /index.html;
-	}
+    add_header Content-Security-Policy "frame-ancestors 'self';" always;
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options nosniff;
+    add_header X-Permitted-Cross-Domain-Policies none;
+
+    index index.html index.htm;
+    charset utf-8;
+
+    gzip on;
+    gzip_disable "msie6";
+
+    gzip_vary on;
+    gzip_proxied any;
+    gzip_comp_level 6;
+    gzip_buffers 16 8k;
+    gzip_http_version 1.1;
+    gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript application/javascript;
+
+    location /next {
+        rewrite ^ /next/ last;
+    }
+
+    location /next/ {
+        alias /opt/graviteeio/apim/portal-ui/next/browser/;
+        try_files $uri $uri/ /next/index.html;
+        sub_filter '<base href="/"' '<base href="/next/"';
+        sub_filter_once on;
+    }
+
+    location / {
+        try_files $uri $uri/ /index.html =404;
+        root /opt/graviteeio/apim/portal-ui;
+        sub_filter '<base href="/"' '<base href="/"';
```

Comment:
Add a comment to explain how to configure the portal_base_href.
See: https://github.com/gravitee-io/gravitee-api-management/blob/8c8bced397a0ce7b0244bf1d444deedc49c33eb4/gravitee-apim-portal-webui/docker/config/default.conf#L40

For instance if portal is served by: https://my-apim-domain/portal
instead of https://portal.my-apim-domain/